content/about/duje/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: About - Duje Schröder
+weight: 10
+---
+
+<div class="text-center mb-5">
+    <img
+        src="https://avatars.githubusercontent.com/u/23352538?v=4"
+        width="150"
+        class="rounded-circle mt-3"
+    />
+    <h3 class="m-3">Duje Schröder</h3>
+    <p>Assistant Coordinator</p>
+</div>
+
+<div class="text-center">
+   _,--',     _,_._.--._______
+ .--.--';_'-. ',_";_      _.,-'
+.'--'.  _.'    {`'-;_ .-...'
+      '-:_      )  / `' '=.
+        ) >     {_/     /.)..
+        |/               `..; .'
+</div>
+
+
+End? No, the journey doesn't end here. MapLibre is another path. One that we
+all must take. On 2020-12-08 22:42:56 UTC MapLibre restarted mapping with its
+mission to provide freedom and independence. Eventually, this is what everybody
+is seeking, from a little project to enterprise endeavours. I am Duje and I am
+assisting as a Coordinator to help maintain what the community has built so
+carefully which means to me:
+
+- Assure MapLibre's fundraising and a healthy Sponsorship Program to keep
+  MapLibre on the map
+- Listen to and discuss with the vivid community which corner to turn next
```

Comment:
```suggestion
- Listen to and discuss with the vibrant community which corner to turn next
```

English nitpick.

"vivid" usually is used with colors; "vibrant" is used with colors and people. :)